Missionframework/modules/04_build/fnc/fn_build_handleRotation.sqf

```diff
@@ -5,17 +5,18 @@
     File: fn_build_handleRotation.sqf
     Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
     Date: 2018-10-07
-    Last Update: 2018-10-08
+    Last Update: 2018-11-04
     License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
 
     Description:
-    -
+        Handles object rotation dragging
 
     Parameter(s):
-    NONE
+        _anchorObject   - Rotation dragging anchor object                       [OBJECT, defaults to objNull]
+        _updatePos      - Rotation should be finished and positions updated     [BOOL, defaults to false]
 
     Returns:
-    NOTHING
+        NOTHING
```

Comment:
Despite it has 2 exitWith, we should try to return BOOL. Or is there anything which I'm missing in this particular function?